src/test/java/com/revature/app/service/VisualizationServiceTest.java

```diff
@@ -1,5 +1,56 @@
 package com.revature.app.service;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.revature.app.dao.VisualizationDao;
+import com.revature.app.model.Curriculum;
+import com.revature.app.model.Visualization;
+
+@ExtendWith(MockitoExtension.class)
 public class VisualizationServiceTest {
+	
+	@InjectMocks
+	
+	private static VisualizationDao mockVisualizationDao;
+	
+	@Autowired
```

Comment:
If this test class is dedicated to unit testing only, @Autowired is not required since we are not using dependency injection w/ an actual Spring container. @InjectMocks should be annotated on the VisualizationService as well.